modules/fundamental/src/advisory/endpoints/mod.rs

```diff
@@ -44,6 +52,7 @@ pub struct ApiDoc;
 
 #[utoipa::path(
     tag = "advisory",
+    context_path = "/api/v1/advisory",
```

Comment:
This is brittle and error-prone, imo.

Reply:
It you have a better idea, I am open for suggestions.